src/laminar.cpp

```diff
@@ -789,18 +803,16 @@ R"x(
     return true;
 }
 
+// TODO: deprecate
 std::string Laminar::getCustomCss() {
     KJ_IF_MAYBE(cssFile, fsHome->tryOpenFile(kj::Path{"custom","style.css"})) {
+        static bool warningShown = false;
+        if(!warningShown) {
+            LLOG(WARNING, "Custom CSS has been deprecated and will be removed from a future release. Use a custom HTML template instead.");
```

Comment:
Is there a reason for this? I know you can inline css in a custom html file, but that's generally frowned on in the web development world.

Reply:
It's frowned upon because inline css (a) can't be shared between multiple pages, (b) can't have different caching rules from the pages that use it, (c) can't benefit from parallel connections and (d) is arguably more difficult to maintain. Since the Laminar frontend is only a single html file, (a) and (b) don't apply; since it's anyway single threaded neither does (c) and (d) is IMO too weak to justify the duplication of functionality.

There's nothing preventing you using a separate css file and serving it from your reverse proxy or other service as recommended in the manual. I want to keep the feature base targeted and succinct to (a) keep Laminar easy to learn and (b) reduce potential attack surface.

Reply:
:+1: works for me.